MAUI/Autocomplete/Selection.md

```diff
@@ -183,7 +183,7 @@ private async void OnSelectionChanged(object sender, Syncfusion.Maui.Inputs.Sele
 
 The following gif image illustrates the result of the above code:
 
-![.NET MAUI Autocomplete](Images/Selection/SelectionChangedEventNotification.png)
+![.NET MAUI Autocomplete selection changed event notification.](Images/Selection/net-maui-autocomplete-selection-changed.png)
```

Comment:
"selection-changed-notification" is the correct term. So, include the notification word in the image name.

Reply:
changed